Documentation/app-container.md

```diff
@@ -33,8 +33,8 @@ To validate that `rkt` successfully implements the ACE part of the spec, use the
 # rkt --insecure-options=image run \
 	--mds-register \
 	--volume=database,kind=host,source=/tmp \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-main.aci \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-sidekick.aci
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-main.aci \
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-sidekick.aci
```

Comment:
This is being taken care of: https://github.com/appc/spec/pull/646
0.8.5 and 0.8.6 artifacts are now both in place. May you reconsider?